### DISCUSSION TITLE
Upload images to folder instead of S3?

### DISCUSSION THREAD

**Discussion by ThreeCO** [memoetapp/memoet#49] [Ideas]
The whole concept of self-hosting memoet, which by the way is a really cool app!, for me is to keep all my content to myself.
Thus I would love to have a switch to decide whether images should go to amazon s3 vs. being stored locally.

**Concept / Idea**
* ENV-Variable working as a switch: UPLOAD_FILES_TO
* accepted values "s3" and "local"

Would that be a feature you would officially implement?

**Comment by manhtai** [comment]
Sure, I used a lib called Waffle to support uploading assets. It has built-in support for local storage and S3 so it can be tweaked to work with local folders easily. I will create an issue for this for later.

Link: #50
